src/fastmcp/server/server.py

```
@@ -1195,194 +1081,201 @@ def disable(
+                    fastmcp_context=ctx,
                 )
-                return list(
-                    await self._run_middleware(
-                        context=mw_context,
-                        call_next=lambda context: self.get_resources(
-                            run_middleware=False
-                        ),
-                    )
+                return await self._run_middleware(
+                    context=mw_context,
+                    call_next=lambda context: self.get_resources(run_middleware=False),
```

Ruff (0.14.13)

Unused lambda argument: `context`

(ARG005)